### PR TITLE
build: Replace virtualenv with venv

### PR DESCRIPTION
**Pull request by loewenheim**
I also slightly changed the beginning of the `Makefile` to unify it with the one in `symbolicator`.

When I run `make test` locally, quite a lot of tests fail, is that expected?

#skip-changelog

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,5 @@
-SYMBOLIC_PYTHON := python3
+SHELL = /bin/bash
+export SYMBOLIC_PYTHON_VERSION := python3
```
Comment (Swatinem) [inline Makefile:2]: this is unused now?

Comment (loewenheim) [reply Makefile:2]: The `SHELL` is probably unnecessary; I did have to add the `export` to get `venv` to work.

File: Makefile [hunk 2]
```diff
@@ -92,5 +93,4 @@ format-python: .venv/bin/python
 
 .venv/bin/python: Makefile
 	@rm -rf .venv
-	@which virtualenv || sudo pip install virtualenv
-	virtualenv -p $(SYMBOLIC_PYTHON) .venv
+	python3 -m venv .venv
```
Comment (jan-auer) [inline Makefile:96]: Did you intend to use `$SYMBOLIC_PYTHON_VERSION` here?

```suggestion
	$SYMBOLIC_PYTHON_VERSION -m venv .venv
```

Comment (loewenheim) [reply Makefile:96]: Yes, I tested it with a hardcoded value before and forgot to change it back >.>

File: Makefile (hunk 1 above)
Comment (loewenheim) [reply Makefile:2]: Sorry, I think I misunderstood you. @jan-auer has pointed out that I forgot to use the `$SYMBOLIC_PYTHON_VERSION` variable.

Comment (jan-auer) [reply Makefile:2]: Oh, my example was for when you define it without the `export`, but you can keep that too. So what do we need the `SHELL` for?

Comment (loewenheim) [reply Makefile:2]: I don't think it's needed for anything, to be honest. Initially I couldn't get it to work and just copied the first two lines from the symbolicator Makefile. I'll remove it.

**Review by jan-auer — APPROVED** [review]
🚢  it

**Pull request merged**